roles/matrix-server/templates/riot-web/config.json.j2

```diff
@@ -10,10 +10,24 @@
     "integrations_jitsi_widget_url": "{{ matrix_riot_web_integrations_jitsi_widget_url }}",
     "bug_report_endpoint_url": "https://riot.im/bugreports/submit",
     "enableLabs": true,
+{% if riot_roomdir_servers is defined %}
+{% for server in riot_roomdir_servers %}
+    "roomDirectory": {
+        "servers": [
+            "{{ server }}"
+        ]
+    },
+{% endfor %}
+{% else %}
```

Comment:
We won't need such `else` clauses if we had defined `riot_roomdir_servers` with a nice default of `['matrix.org']`.